std/algorithm/comparison.d

```diff
@@ -211,6 +211,13 @@ Throws: If none of the choice matches, a $(D SwitchError) will be thrown.  $(D
 SwitchError) will also be thrown if not all the choices are void and a void
 choice was executed without throwing anything.
 
+Params:
+    choices = The choices to select from.
```

Comment:
`choices_` looks better in prose like this. Otherwise it looks like some kind of cyclical definition :)